<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFMainForm</class>
 <widget class="QMainWindow" name="QFMainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QFMainForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item row="0" column="2">
        <widget class="QLineEdit" name="QLEServer">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Socket</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Subject</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="QLESubject">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Server</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="QSBSocket">
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>25</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>From</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLineEdit" name="QLEFrom">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>To</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLineEdit" name="QLETo">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Cc</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Bcc</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QLineEdit" name="QLEBcc">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Body</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Attach</string>
         </property>
        </widget>
       </item>
       <item row="14" column="2">
        <widget class="QLineEdit" name="QLEAttach">
         <property name="text">
          <string>a.jpg</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>UserID</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="QCBAuthentication">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Authentication</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Priority</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QRadioButton" name="QRBNormal">
            <property name="text">
             <string>Normal</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="QRBMedium">
            <property name="text">
             <string>Medium</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="QRBHigh">
            <property name="text">
             <string>High</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="13" column="2">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Text Plain</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item>
            <widget class="QPlainTextEdit" name="QPTEBodyPlain">
             <property name="plainText">
              <string>First row
Second row
Third row</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Text Html</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item>
            <widget class="QPlainTextEdit" name="QPTEBodyHtml">
             <property name="plainText">
              <string>&lt;body&gt;&lt;b&gt;First row&lt;/b&gt;
&lt;br&gt;Second row
&lt;br&gt;Third row
&lt;br&gt;&lt;img src=&quot;cid:b&quot; /&gt;
&lt;/body&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QCheckBox" name="QCBHtmlFormat">
         <property name="text">
          <string>Html format</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QLineEdit" name="QLECc">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QCheckBox" name="QCBSSLConnection">
         <property name="text">
          <string>SSL connection</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="QLEUserID">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="QLEPassword">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>7</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QPushButton" name="QPBSend">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="QPBClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>QLEServer</tabstop>
  <tabstop>QSBSocket</tabstop>
  <tabstop>QCBAuthentication</tabstop>
  <tabstop>QCBSSLConnection</tabstop>
  <tabstop>QLEUserID</tabstop>
  <tabstop>QLEPassword</tabstop>
  <tabstop>QLESubject</tabstop>
  <tabstop>QLEFrom</tabstop>
  <tabstop>QLETo</tabstop>
  <tabstop>QLECc</tabstop>
  <tabstop>QLEBcc</tabstop>
  <tabstop>QRBNormal</tabstop>
  <tabstop>QRBMedium</tabstop>
  <tabstop>QRBHigh</tabstop>
  <tabstop>QCBHtmlFormat</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>QPTEBodyPlain</tabstop>
  <tabstop>QPTEBodyHtml</tabstop>
  <tabstop>QLEAttach</tabstop>
  <tabstop>QPBSend</tabstop>
  <tabstop>QPBClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
